{"name": "test_login[13812347863-123456-8888]", "status": "failed", "statusDetails": {"message": "AssertionError: assert 'http://192.168.249.129/index.php/Home/user/login.html' == 'http://192.168.249.129/index.php/Home/User/index.html'\n  \n  - http://192.168.249.129/index.php/Home/User/index.html\n  ?                                       ^      ---\n  + http://192.168.249.129/index.php/Home/user/login.html\n  ?                                       ^    +++", "trace": "self = <scripts.test_pytest_login.TestLogin object at 0x000001C47AAA24A0>\n\n    def teardown_method(self):\n        # # 断言\n        print(self.driver.current_url)\n>       assert self.driver.current_url == \"http://192.168.249.129/index.php/Home/User/index.html\"\nE       AssertionError: assert 'http://192.168.249.129/index.php/Home/user/login.html' == 'http://192.168.249.129/index.php/Home/User/index.html'\nE         \nE         - http://192.168.249.129/index.php/Home/User/index.html\nE         ?                                       ^      ---\nE         + http://192.168.249.129/index.php/Home/user/login.html\nE         ?                                       ^    +++\n\ntest_pytest_login.py:36: AssertionError"}, "parameters": [{"name": "username", "value": "'13812347863'"}, {"name": "password", "value": "'123456'"}, {"name": "verify_code", "value": "'8888'"}], "start": 1721230959425, "stop": 1721230959517, "uuid": "d881fb9b-79b5-4fe0-8923-477e499ccf19", "historyId": "62f5effa1ec95ef1a4ddbfb27d58c35c", "testCaseId": "271a2d32072a5453558e7527f142fa03", "fullName": "scripts.test_pytest_login.TestLogin#test_login", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_pytest_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Lin"}, {"name": "thread", "value": "23756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_pytest_login"}]}